resources/js/src/app/components/itemList/filter/ItemRangeSlider.vue

```diff
@@ -58,7 +58,8 @@ import { FacetAware, TemplateOverridable } from '../../../shared/interfaces';
 import { computed, defineComponent, onMounted, ref, watch } from '@vue/composition-api';
 import UrlBuilder, { PriceFacetValue } from '../../../shared/UrlBuilder';
 import TranslationService from '../../../shared/TranslationService';
-import * as noUiSlider from 'noUiSlider';
+import * as noUiSlider from 'nouislider';
+import 'nouislider/dist/nouislider.css';
```

Comment:
Would it be simpler to include it within the `<style>` block? Maybe the test adaptions for the styling would not be needed anymore.

Reply:
Maybe I need to clarify, that I don't want you to copy the file content, just something like this:
```
<style scoped lang="scss">
@import 'nouislider/dist/nouislider.css'
</style>
```
I think it helps the readability, as the code and the styles are clearly separated.

And maybe it's even needed, with my suggestion from https://github.com/findologic/plugin-plentymarkets-ceres-api/pull/348#discussion_r1082461662. Except we find another solution for that.